backend/users/views.py

```diff
@@ -0,0 +1,38 @@
+from django.shortcuts import render
+
+# Create your views here.
+
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from .models import User
+from .serializers import UserSerializer
+
+
+@api_view(['GET'])
+def user_detail(request, pk, format=None):
+    """
+    Retrieve or update an order.
+    """
+    token = request.GET.get('token')
+    if token is None or token == '':
+        return Response(status=status.HTTP_400_BAD_REQUEST)
+
+    try:
+        user = User.objects.get(pk=pk)
+    except User.DoesNotExist:
+        return Response(status=status.HTTP_404_NOT_FOUND)
+
+    if request.method == 'GET':
+        serializer = UserSerializer(user)
+        return Response(serializer.data)
+
+
+@api_view(['POST'])
+def register(request, format=None):
```

Comment:
I think, that is obvious that only register app in our backend can trigger this endpoint (creation of new user == register new user). So we should somehow forbid this action for all users except register module. But for now this is not a big problem, security is out of MVP, so if you dont know how to do it, you can create an issue with low priority and forget about this problem for some time